src/main/java/frc/robot/Robot.java

```diff
@@ -35,9 +35,25 @@
   public void robotInit() {
     pdp = new LoggablePowerDistribution(1, ModuleType.kRev);
     
-    driver = new LoggableController("Driver", 0);
+    climber = new LoggableController("Climber", 0);
     operator = new LoggableController("Operator", 1);
 
+    if(this.climberEnabled) {
+      System.out.println("Initializing climber...");
+      ClimbingSensors touchA = new ClimbingSensors("TouchA", 0, 1);
+      ClimbingSensors touchB = new ClimbingSensors("TouchB", 0, 1);
+      ClimbingSensors touchC = new ClimbingSensors("TouchC", 0, 1);
+      climb = new ClimbingSensors(touchA, touchB, touchC);
```

Comment:
climb object should be Climbing class and have (9, 10, 2, 3, 4, 5, touchA, touchB, touchC)